tools/unit_tests.py

```diff
@@ -48,7 +50,6 @@ def unit_tests(deno_exe):
     run_unit_test(deno_exe, "permR0W1N0E0U0", ["--allow-write"])
     run_unit_test(deno_exe, "permR1W1N0E0U0",
                   ["--allow-read", "--allow-write"])
-    run_unit_test(deno_exe, "permR1W0N1E0U0", ["--allow-read", "--allow-net"])
```

Comment:
Is this intentional?

Reply:
Yes - there are zero tests that match this and so it triggers the new assert.